hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/OMConfigKeys.java

```diff
@@ -270,6 +270,13 @@ private OMConfigKeys() {
 
   public static final String OZONE_OM_METADATA_LAYOUT_PREFIX = "PREFIX";
 
+  // Defines the default bucket behaviour when OM does not pass the
```

Comment:
please modify comment to:

`Default bucket layout used by Ozone Manager during bucket creation when a client does not specify the bucket layout option.`